scripts/dts/gen_defines.py

```diff
@@ -98,6 +98,9 @@ def main():
             out_comment(f"Node's full path:")
             out_dt_define(f"{node.z_path_id}_PATH", f'"{escape(node.path)}"')
 
+            out_comment(f"Node's name:")
```

Comment:
```suggestion
            out_comment("Node's name:")
```

Reply:
@gmarull thanks.  I stole the pattern from a few lines above.